pkg/registry/rbac/validation/rule_test.go

```diff
@@ -76,7 +76,7 @@ func TestDefaultRuleResolver(t *testing.T) {
 	staticRoles1 := StaticRoles{
 		roles: []*rbacv1.Role{
 			{
-				ObjectMeta: metav1.ObjectMeta{Namespace: "namespace1", Name: "readthings"},
+				ObjectMeta: metav1.ObjectMeta{Namespace: "namespace1", Name: "readthings", Tenant: metav1.TenantSystem},
```

Comment:
we should add a non-system tenant case

Reply:
will file a PR after this one is merged.